test/unit/spec/util/asyncTools.js

```diff
@@ -0,0 +1,59 @@
+define([], () => {
+    /**
+     * A module providing functions to help with asynchronous tests.
+     * These functions are general purpose - they are not tied to any test framework or library.
+     */
+
+    'use strict';
+
+    const TRY_LOOP_INTERVAL = 100;
+
+    /**
+     * Returns a promise which resolves when the given duration, in milliseconds,
+     * elapses.
+     *
+     * @param {int} duration
+     * @returns {Promise<void>}
+     */
+    function waitFor(duration) {
```

Comment:
This is effectively a duplicate of https://github.com/kbase/narrative/blob/develop/test/unit/testUtil.js#L63 and should just use that one.

The `tryFor` function could probably live in `testUtil` as well. At the least, this file should be moved out of the `test/spec/util` directory and up into the  should be `test` directory, as it's not a test spec.

Reply:
Yeah, I think this was done a long time ago in the original PR, and I was trying to make as few changes from that as possible. E.g. there was nothing else in the util directory then, just mwsTool.js!
Knowing me, I also wanted the tests to be low-dependency, and it seemed like there as a lot going on in testUtils...
Anyway, I'm happy to move the code out and into testUtils (I had moved the non-spec files out myself during testing so I could isolate tests w/o the util specs.)

Reply:
Okay, have the changes locally, will update the PR a bit later this afternoon.

Reply:
So the changes are up. There were changes to a dozen or so files to enable this, and I included some commented-out test config features that I've found useful for focusing tests and reducing test output clutter.
The mswUtils was moved into a unit/utils directory to clarify purpose and ease inclusion of test utilities (e.g. don't need to individually add that file, don't need to individually map it for requires). asyncUtils was removed and the non-duplicate function (tryFor) moved into testUtils.